userguide/getting-started/quick_start_with_kb_api.adoc

```diff
@@ -413,9 +431,13 @@ $pm = $accountApi->createPaymentMethod(
 </div>
 ++++
 
-==== Creating Your First Subscription
+[step3]
+==== Set Up a Subscription for the Account
+
+You are now ready to create a https://docs.killbill.io/latest/Kill-Bill-Glossary.html#subscription[subscription^] for the customer.
 
-Note: replace `1cb6c8b0-1df6-4dd5-9c7c-2a69bab365e8` below with the ID of your account.
+[NOTE]
+*Note:* Replace `1cb6c8b0-1df6-4dd5-9c7c-2a69bab365e8` below with the ID of your account.
 
```

Comment:
Would also be good to mention that `planName` must match the plan name in the catalog.

Reply:
Fixed

Reply:
Done